app/src/main/java/com/kickstarter/libs/Koala.java

```diff
@@ -344,9 +345,14 @@ public void trackSavedPaymentMethod() {
     AnswersExtKt.fabricLogCustomEvent(KoalaEvent.SAVED_PAYMENT_METHOD);
   }
 
-  public void trackSelectedChosenCurrency() {
-    this.client.track(KoalaEvent.SELECTED_CHOSEN_CURRENCY);
-    AnswersExtKt.fabricLogCustomEvent(KoalaEvent.SELECTED_CHOSEN_CURRENCY);
+  public void trackSelectedChosenCurrency(final String selectedCurrency) {
+    this.client.track(KoalaEvent.SELECTED_CHOSEN_CURRENCY, new HashMap<String, Object>() {
+      {
+        put("user_chosen_currency", selectedCurrency);
+      }
+    });
+
+    AnswersExtKt.fabricLogCustomEventWithAttributes(KoalaEvent.SELECTED_CHOSEN_CURRENCY, "currency", selectedCurrency);
```

Comment:
I thought we're not doing any more Fabric events. 👀

Reply:
Removed!